app/services/irs_attempts_api/tracker.rb

```diff
@@ -43,7 +43,8 @@ def track_event(event_type, metadata = {})
 
       if enabled?
         if IdentityConfig.store.irs_attempt_api_payload_size_logging_enabled
-          analytics.irs_attempts_api_event_metadata(
+          # analytics may be nil when called from outside ApplicationController
+          analytics&.irs_attempts_api_event_metadata(
```

Comment:
let's find a way to pass in or construct an `Analaytics` instance in this flow. This metadata is useful for our dashboards and it would be bad to have gaps of missed data

Reply:
^ Done, but that makes me think I should be able to remote the nil-safe operator now.